### PR TITLE
Use variables in post-deploy steps

### PR DESCRIPTION
**Pull request by deci-ftw**
Updated `<namespace>` and `<app name>` references with `$namespace` and `$name`
The post deploy UI will expand these to their values, allowing the user to copy the commands easily. 

Additionally, guarded the example commands by making them comments to prevent accidental runs

Preview of next steps after this change:
next-steps-variables

### CODE REVIEW

File: manifest/application.yaml.template [hunk 1]
```diff
@@ -27,20 +27,20 @@ spec:
       Get the CloudBees Core Operations Center URL:
```
Comment (huyhg) [inline manifest/application.yaml.template]: Can you correct deploy-info as well please? The product_id is not accurate.

File: manifest/application.yaml.template [hunk 2]
```diff
@@ -27,20 +27,20 @@ spec:
...
 
       The setup process will prompt you for an intial admin password. Execute this command to get it:
- 
+
       ```shell
-      kubectl exec <app name>-cjoc-0 -n <namespace> -- cat /var/jenkins_home/secrets/initialAdminPassword
+      kubectl exec $name-cjoc-0 -n $namespace -- cat /var/jenkins_home/secrets/initialAdminPassword
 
-      ex. kubectl exec cloudbees-core-1-cjoc-0 -n deployer-test -- cat /var/jenkins_home/secrets/initialAdminPassword
+      # ex. kubectl exec cloudbees-core-1-cjoc-0 -n deployer-test -- cat /var/jenkins_home/secrets/initialAdminPassword
```
Comment (huyhg) [inline manifest/application.yaml.template]: I think this can be removed now.

File: manifest/application.yaml.template (hunk 1 above)
Comment (deci-ftw) [reply manifest/application.yaml.template]: Done.

File: manifest/application.yaml.template (hunk 2 above)
Comment (deci-ftw) [reply manifest/application.yaml.template]: Done.

**Pull request merged**